pkg/cmd/pulumi/packagecmd/package_publish.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016-2024, Pulumi Corporation.
+// Copyright 2025, Pulumi Corporation.
```

Comment:
It's really unfortunate that this shows up as a modification. In reality the original file has been renamed to `package_publish_sdk.go` :(